src/core/qgsvectorlayer.h

```diff
@@ -2482,6 +2482,12 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
     //! Emitted before changes are rolled back.
     void beforeRollBack();
 
+    /**
+     * Emitted after changes are commited to the data provider.
+     * \since QGIS 3.16
+     */
+    void afterCommitChanges();
```

Comment:
```suggestion
    void changesCommitted();
```

Reply:
I chose this name, as there is already existing `beforeCommitChanges`, so it's natural to name it `afterCommitChanges` to me? If you don't follow my logic, I am fine to change it as suggested.

Reply:
all good